rust-runtime/smithy-json/src/escape.rs

```diff
@@ -70,10 +70,9 @@ mod test {
     proptest! {
         #[test]
         fn matches_serde_json(s in ".*") {
-            assert_eq!(
-                serde_json::to_string(&s).unwrap(),
-                format!(r#""{}""#, escape_string(&s))
```

Comment:
you probably want to enable rustfmt on save

Reply:
Looks like the proptest macro hid the assert_eq from the formatter.